config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -0,0 +1,67 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
This file is only needed to enable 6.1 defaults one at a time. If possible can you test each one and then remove this and enable the 6.1 defaults?

Reply:
Want to ship this and let it run for a bit first, so we have a smaller search space for any issues we hit. Then will swap over the defaults one by one!